geosys/bridge_api/default.py

```diff
@@ -47,6 +47,7 @@
 CLIENT_SECRET = 'mapproduct_api.secret'
 GRANT_TYPE = 'password'
 SCOPE = 'openid offline_access'
+MAX_FEATURE_NUMBERS = None
```

Comment:
@VincentLel do you have any idea what is the maximum number of features to be processed?

Reply:
default to 10 for now @myarjunar pending feedback from @VincentLel - will be good to indicate in the UI what the limit is too so things are transparent to the user.